src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -596,31 +605,38 @@ void submitStreamReadOperation(final PathOperation streamOp, final String nodeAd
 		// TODO: Alex, issue SDP-51
 	}
 
+	boolean completeStreamDeleteOperation(final PathOperation streamOp, final boolean result, final Throwable thrown,
```

Comment:
Couldn't we get the scope/stream names from the `streamOp`? Just to not to pass extra arguments